docs/querying/sql-functions.md

```diff
@@ -1154,11 +1154,29 @@ Returns the following:
 
 ## GREATEST
 
-`GREATEST([expr1, ...])`
+Returns the maximum value from the provided expressions. The expressions must all be convertible to a common data type.
```

Comment:
What does "convertible to a common data type" mean? Convertible by whom? By Druid? Data type common to Druid? I think this sentence could use some clarification.